src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/OpprettFagsakModal.tsx

```diff
@@ -73,13 +103,70 @@ const OpprettFagsakModal: React.FC<IOpprettFagsakModal> = ({
         lukkModal();
     };
 
+    useEffect(() => {
+        if (bruker === undefined && søkeresultat) {
+            hentPerson(søkeresultat.ident).then(personRessurs => {
+                if (personRessurs.status === RessursStatus.SUKSESS) {
+                    settBruker(personRessurs.data);
+                }
+            });
+        }
+    }, [søkeresultat]);
+
     useEffect(() => {
         if (samhandlerSkjema.submitRessurs.status === RessursStatus.SUKSESS) {
             settValgtSamhandler(samhandlerSkjema.submitRessurs.data);
             settVisFeilmelding(false);
         }
     }, [samhandlerSkjema.submitRessurs.status]);
 
+    const valgAvFagsakType = () => (
+        <>
+            <StyledFagsakOptionsDiv>
+                <FamilieReactSelect
```

Comment:
Her ser det ut som at skissene egentlig ber om en `Select` fra `@navikt/ds-react`, er det en grunn til at vi heller bruker denne? `FamilieReactSelect` brukes gjerne når vi vil ha multiselect (for det støtter ikke designsystemet 😢 ) og når vi skal støtte lesevisning

Reply:
Leter som regel først blant komponenter som finnes i løsningen fra før, og blant de var det FamilieReactSelect som lignet mest 😄 Men jeg kan godt prøve å ta i bruk `Select` fra `@navikt/ds-react` 👍